CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 * Remove Tax/Vat column for Manual Fees/Fines Table. Fixes UIU-577.
 * Ensure the availability of data to Charge manual Fee/Fine. Fixes UIU-219.
 * Add functions for link to fee/fine history and fee/fine details. Fixes UIU-238 and UIU-239.
-* Bump circulation interface to v.4.0. Part of UIU-627.
+* Support `circulation` interface 3.0 or 4.0. Part of UIU-627.
+* Support `loan-storage` interface 4.0 and 5.0. Part of UIU-627.
```

Comment:
Why "or" for the previous entry but "and" for this one? Do you mean something different in the two cases?

Reply:
No, that is my rushing and should be corrected. 

The intent is to state that either is supported (and clearly I failed to know how to say that well).